user-guide/network/nictag.rst

```diff
@@ -11,6 +11,10 @@ For other use cases, it is possible to add following NIC tags:
 - internal
 - storage
 
+Administrator can also add new NIC tags (e.g. overlay, etherstub), these new NIC tags are then collected from the node and will be presented when creating network.
+
+.. warning:: Overlay and etherstub NIC tags are created manually and it is the administrator's job to make sure that different nodes don't have NIC tags with the same name but different type. Removing one of the NIC tags with different type or setting the same type on NIC tag on all nodes will solve this issue.
+
 .. note:: **external**, **internal**, **storage** NIC tags are preconfigured in the *Danube Cloud* system. However, it is up to the data center administrator to do the proper configuration on the compute nodes.
```

Comment:
- Suggestion: "...are preconfigured in the *Danube Cloud* install script."  (because it's not true they are preconfigured in the DC itself)
- but I don't quite understand the overall purpose of this note in the light of upcoming changes